src/main/resources/application.properties

```diff
@@ -2,4 +2,7 @@ server.port = 8081
 spring.cloud.gcp.project-id=absolute-text-251105
 spring.cloud.gcp.credentials.location=file:/app/config/secret.json
 location.api.fetch.nearby.users=http://localhost:8080/location/user
-notification.api.broadcast=http://localhost:8086/notification/broadcast
\ No newline at end of file
+notification.api.broadcast=http://localhost:8086/notification/broadcast
+users.api.usersWithInterest=http://localhost:8083/userwithinterest
```

Comment:
We should always have /user/ as part of the API resource depicting which service we are calling, from the URL. If this resource has already been implemented on User service with this url, we should update User service as well.